src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -753,7 +753,9 @@ def import_key(cmd, client, vault_base_url, key_name, protection=None, key_ops=N
             try:
                 pkey = load_pem_private_key(pem_data, pem_password, default_backend())
             except (ValueError, TypeError, UnsupportedAlgorithm) as e:
-                print(e)
+                if str(e) == 'Could not deserialize key data.':
```

Comment:
Lower()?

Reply:
> Lower()?

@yungezz For this case we only need an exact match, this error message was raised by crypto lib, I copied the message directly.